x/tokenfactory/keeper/genesis.go

```diff
@@ -0,0 +1,51 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/desmos-labs/desmos/v5/x/tokenfactory/types"
+)
+
+// InitGenesis initializes the tokenfactory module's state from a provided genesis
+// state.
+func (k Keeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
+	k.createModuleAccount(ctx)
+
+	k.SetParams(ctx, genState.Params)
+
+	for _, genDenom := range genState.GetFactoryDenoms() {
+		creator, _, err := types.DeconstructDenom(genDenom.GetDenom())
+		if err != nil {
+			panic(err)
+		}
+		err = k.createDenomAfterValidation(ctx, creator, genDenom.GetDenom())
+		if err != nil {
+			panic(err)
+		}
+		err = k.SetAuthorityMetadata(ctx, genDenom.GetDenom(), genDenom.GetAuthorityMetadata())
+		if err != nil {
+			panic(err)
+		}
+	}
+}
+
+// ExportGenesis returns the tokenfactory module's exported genesis.
+func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
+	genDenoms := []types.GenesisDenom{}
+	iterator := k.GetAllDenomsIterator(ctx)
+	defer iterator.Close()
+	for ; iterator.Valid(); iterator.Next() {
```

Comment:
Let's add a blank line here before the `foor` loop so that it's easier to read

Reply:
Added